keyboards/keebfront/vanguard65/keymaps/via/keymap.c

```diff
@@ -0,0 +1,252 @@
+/*
+Copyright 2022 <hello@keebfront.com>
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include QMK_KEYBOARD_H
+#include "analog.h"
+#include "string.h"
+
+enum layers {
+    _LAYER0,
+    _LAYER1,
+    _LAYER2,
+    _LAYER3
```

Comment:
```suggestion
    _LAYER1
```

Reply:
@waffle87 I can't remove these as the intent is to have 4 layers available for use, even though they are blank currently. I cannot _add_ additional layers once it has been compiled, and I cannot have the users recompile it to make use of the core functionality of the board. The **slider** is coded currently to switch between Layers 1-4 as a feature.